### DISCUSSION TITLE
Point Grey FlyCapture stoped working

### DISCUSSION THREAD

**Discussion by NunoMac** [bonsai-rx/bonsai#1730] [Q&A]
I had FlyCapture working until a week ago and it stopped working. This is the situation:
I have the Pointgrey package installed in bonsai, and the Pointgrey Flycap2 installed in the Windows. 
Flycap2 can open the camera.
Bonsai has the Point Grey packages installed and the FlyCapture node is available from the menu. But when trying to add the node to the protocol, there is an error: `The specific type could not be found. Parameter name: typeName`
Other nodes seem to be able to be created normally.
The same problem started happening on a Bonsai portable folder too.
I reinstalled the package but the problem persists.

I found a similar problem here: https://github.com/bonsai-rx/bonsai/issues/658
Is uninstalling Bonsai and Point Grey completely and install again the only solution? (assuming that works)

Versions below:
Windows 11 Pro build 22631.3296
FlyCap 2.16.3.0
Bonsai 2.8
PointGrey Library 2.3.1

I would greatly appreciate some help.

Best,

Nuno

**Comment by bruno-f-cruz** [comment]
Hey,
Did you try installing the FlyCapture version mentioned in the post you cited? (i.e. `FlyCap2Viewer_2.11.3.425_x64`). In general I strongly advise you to move away from the FlyCap package (deprecated) and use the SpinnakerCapture one instead.

**Reply by NunoMac** [reply]
Hello,

Problem solved. Had to reinstall both Bonsai and Flycap2 (reinstalling FlyCap2 probably enough)

> Did you try installing the FlyCapture version mentioned in the post you cited? (i.e. FlyCap2Viewer_2.11.3.425_x64)
Yes, that is the version.

>strongly advise you to move away from the FlyCap package (deprecated) and use the SpinnakerCapture one instead.
Would the latest SpinnakerCapture version work?
Right now I have some old protocols that I have to rewrite in more recent versions to test that...

On a side note, trying to understand Bonsai installations,
after reinstalled Bonsai before reinstalling FlyCap2 some things I was not expecting happened:
- on an older Bonsai ZIP version, the FlyCapture option disappeared too!
- after reinstalling Bonsai (on root) the node appears on root Bonsai (still not working as before) but did not reappear on the ZIP version
- - checking the Packages folder in ZIP version, the Bonsai.PointGrey.2.3.1 files are there!
I definitely did not expect the main root installation to affect the ZIP versions

Thank you for the tips!

Nuno
